src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Runtime.pkgproj

```diff
@@ -16,4 +14,12 @@
   <Import Condition="'$(ExcludeHostAssets)' != 'true'" Project="..\..\Microsoft.NETCore.DotNetHostPolicy\Microsoft.NETCore.DotNetHostPolicy.props" />
   <Import Condition="'$(ExcludeHostAssets)' != 'true'" Project="..\..\Microsoft.NETCore.DotNetHostResolver\Microsoft.NETCore.DotNetHostResolver.props" />
 
+  <!-- Keep this isolated to desktop mono for now -->
+  <Target Name="MobileGetBuildRidSpecificPackageProps"
+          Condition="'$(RuntimeFlavor)' == 'Mono' and '$(TargetsMobile)' == ''"
+          AfterTargets="GetBuildRidSpecificPackageProps">
+    <PropertyGroup>
+      <RidSpecificPackProperties>BaseId=$(MSBuildProjectName)$(RuntimeSpecificFrameworkSuffix).Mono.$(PackageBuildRID);IdPrefix=</RidSpecificPackProperties>
```

Comment:
I guess my suggestion wasn't quite clear... this should be done with a small tweak over here (but without the redundant `.Mono`):

https://github.com/dotnet/arcade/blob/be7971c98d4ad60cdf8c3d1aa41f950f42bd466a/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/framework.packaging.targets#L224

```
<RidSpecificPackProperties>BaseId=$(MSBuildProjectName)$(RuntimeSpecificFrameworkSuffix).$(PackageBuildRID);IdPrefix=</RidSpecificPackProperties>
```

And you'd use that API here in dotnet/runtime by specifying the property:

```xml
<PropertyGroup Condition="'$(RuntimeFlavor)' == 'Mono' and '$(TargetsMobile)' == ''">
  <RuntimeSpecificFrameworkSuffix>.Mono</RuntimeSpecificFrameworkSuffix>
</PropertyGroup>
```

Adding a target here that forces the value reduces the flexibility of changes in the sharedfx SDK later on. (If you'd like to just go ahead with this due to time pressure, please at least file an issue to track removing it.)

Reply:
I'll file a tracking issue.

Reply:
https://github.com/dotnet/runtime/issues/35107